keyboards/handwired/42/mcuconf.h

```diff
@@ -0,0 +1,273 @@
+/*
```

Comment:
Same deal with this file, but because there are no guards we need to undefine/redefine:

```c
<license header>
#pragma once
#include_next "mcuconf.h"
#undef STM32_I2C_USE_I2C1
#define STM32_I2C_USE_I2C1 TRUE
```